app/models.py

```diff
@@ -679,9 +679,10 @@ class ServiceInboundApi(db.Model, Versioned):
 
     @property
     def bearer_token(self):
-        if self._bearer_token:
-            return encryption.decrypt(self._bearer_token)
-        return None
+        # Column is non-nullable
+        # if self._bearer_token:
+        return encryption.decrypt(self._bearer_token)
+        # return None
```

Comment:
If the column must have a value at this point, I say we just eliminate the commented out lines.  Thanks for flagging!